.golangci.yml

```diff
@@ -379,6 +383,22 @@ issues:
         # - unparam
         # - wsl
 
+    - path: config/configurations.go|main.go
+      text: "forbidden by pattern"
+      linters:
+        - forbidigo
+
+    # TODO(rigdomin): handler this linter at a later time
+    - path: formatter/markdown.go|handlers/reference/reference.go
+      text: "reassigned, but never used afterwards"
+      linters:
+        - wastedassign
+
+    - path: spec/spec.go
```

Comment:
can you add a comment indicating how far outside of the configured bounds this is for a reference?

Reply:
commented